### PR TITLE
fix(vrack-services): fix vrack services staging

### PR DESCRIPTION
**Pull request by chipp972**
blocking scroll

ref: MANAGER-13171

| Question         | Answer
| ---------------- | ---
| Branch?          | `master` / `release/**` / `develop` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-13171 #... 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

### CODE REVIEW

File: packages/manager/apps/vrack-services/src/components/DeleteModal.tsx [hunk 1]
```diff
@@ -55,22 +55,25 @@ export const DeleteModal: React.FC<DeleteModalProps> = ({
...
     closeModal();
+    modal.current.close();
   };
 
   React.useEffect(() => {
     if (isModalOpen && onDisplayDataTracking) {
       tracking.trackPage({
         name: onDisplayDataTracking,
-        level2: '',
+        level2: '0',
```
Comment (aboungnaseng-ovhcloud) [inline packages/manager/apps/vrack-services/src/components/DeleteModal.tsx:69]: we have to add page category="pop-up"

File: packages/manager/apps/vrack-services/src/index.tsx [hunk 2]
```diff
@@ -53,15 +56,24 @@ const init = async (
...
-  ReactDOM.createRoot(rootElement).render(
-    <React.StrictMode>
-      <ShellProvider client={context}>
-        <App />
-      </ShellProvider>
-    </React.StrictMode>,
-  );
+  const rootElement = document.getElementById('ovh-app') as HTMLElement;
+
+  console.log('root element', rootElement);
```
Comment (aboungnaseng-ovhcloud) [inline packages/manager/apps/vrack-services/src/index.tsx:63]: delete console log

File: packages/manager/apps/vrack-services/src/index.tsx [hunk 3]
```diff
@@ -53,15 +56,24 @@ const init = async (
...
+
+  setTimeout(() => {
+    ReactDOM.createRoot(rootElement).render(
+      <React.StrictMode>
+        <ShellProvider client={context}>
+          <App />
+        </ShellProvider>
+      </React.StrictMode>,
+    );
+    console.log('done render');
```
Comment (aboungnaseng-ovhcloud) [inline packages/manager/apps/vrack-services/src/index.tsx:73]: delete console log

File: packages/manager/apps/vrack-services/src/components/DeleteModal.tsx (hunk 1 above)
Comment (chipp972) [reply packages/manager/apps/vrack-services/src/components/DeleteModal.tsx:71]: like this ?

**Pull request merged**